src/objects/nade.lua

```diff
@@ -4,17 +4,16 @@
 --]]
 
 local obj = require 'obj'
-local nade_width = 16
 local nade_height = 16
 
 return {
     init = function(self)
 	    -- consts
-	    img = love.graphics.newImage('assets/sprites/16x16_nade.png')
-    	self.gravity = self.gravity or 350
+	    self.img = love.graphics.newImage('assets/sprites/16x16_nade.png')
```

Comment:
Should all variables be this way unconditionally? Can we style check for this?

Reply:
Globals in object types are generally undesirable as they are shared between all objects of that type. This is good for sharing resources like images but resource caching should be implemented in another subsystem (maybe an 'assets' interface)

Reply:
To clarify, i'm referring to using `self` for instantiated object variables